charts/honeycomb/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: honeycomb
 description: Honeycomb Kubernetes Agent
-version: 1.3.1
-appVersion: 2.3.2
+version: 1.4.0
```

We prefer to update the chart version as part of the release PR/commit that includes the changelog and is what is tagged.
```suggestion
version: 1.3.1
```